crates/modelardb_server/src/metadata/mod.rs

```diff
@@ -793,7 +793,9 @@ impl MetadataManager {
                     original_expr: None,
                 };
 
-                generated_columns[row.get::<usize, usize>(0)?] = Some(generated_column);
+                // column_index is stored as a i64 as a model table contains at most 1024 columns.
```

Comment:
The comment is to make it clear why we do not perform a bitwise conversion from `u64` to `i64` since SQLite only supports signed integers. I have tried to extend the two comments to make this more clear.

Reply:
OK